deepchem/models/tensorgraph/layers.py

```diff
@@ -1516,6 +1565,35 @@ def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
     return out_tensor
 
 
+class MaxPool1D(Layer):
```

Comment:
Could you add a high level description here?

Reply:
Just added!